.github/workflows/lint.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
           VALIDATE_ALL_CODEBASE: false
           DEFAULT_BRANCH: main
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VALIDATE_PYTHON: true
           VALIDATE_R: true
           VALIDATE_SQLFLUFF: true
-          SQLFLUFF_CONFIG_FILE: /.sqlfluff
+          SQLFLUFF_CONFIG_FILE: ./.sqlfluff
```

Comment:
**[Question, non-blocking]** Does this change work? In my branch to [fix superlinter](https://github.com/ccao-data/data-architecture/pull/37) I had to set `LINTER_RULES_PATH` to `/` because it's [set to `.github/linters` by default](https://github.com/super-linter/super-linter#environment-variables), so I would assume that `./.sqlfluff` would expand out to `.github/linters/.sqlfluff` without the change to `LINTER_RULES_PATH`. Maybe pulling in the latest change from `master` will resolve the SQLFluff problems you were experiencing?